paparazzi-gradle-plugin/src/test/projects/compose-a11y/src/test/java/app/cash/paparazzi/plugin/test/ComposeA11yTest.kt

```diff
@@ -36,6 +37,7 @@ class ComposeA11yTest {
     paparazzi.snapshot(mixedView)
   }
 
+  @Ignore("This test is failing due to LayoutLib 14.0.11 bump.")
```

Comment:
1) any thoughts on what's causing the test to fail?
2) prob best to not merge 14.0.11 before the pending release.

Reply:
yep, found this out and was going to update. 

There was a change in `WindowManagerImpl` when adding the decor window to the base root view. 

in 14.0.9 it calls 
```this.mBaseRootView.addView(layout, this.mBaseRootView.getChildCount() - 1, new FrameLayout.LayoutParams(layoutMode, layoutMode));```

in 14.0.11 it calls
```this.mBaseRootView.addView(layout, new FrameLayout.LayoutParams(layoutMode, layoutMode));```

This causes the material3 bottom sheet test to render unexpectedly. I will close this and revert 14.0.11 bump